storage/responses/google-classroom/google-classroom-student.json

```diff
@@ -0,0 +1,6 @@
+{
+    "student": {
+        "id": "102628873478251109814",
```

Comment:
Is the ID string here?

Reply:
Well, in the response, that is a numeric identifier. 
But for the API call, as per Google API documentation, the userID parameter is a string:
https://developers.google.com/classroom/reference/rest/v1/courses.students/get